openedx/core/djangoapps/content_tagging/rest_api/v1/urls.py

```diff
@@ -30,5 +26,9 @@
         oel_tagging_views_import.TemplateView.as_view(),
         name="taxonomy-import-template",
     ),
+    path(
+        "object_tags/<str:object_id>/export/",
```

Comment:
```suggestion
        "object_tags/<str:context_id>/export/",
```
I think it's weird to call this `object_id` if it only works with courses (and eventually, libraries).